src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
```

Comment:
you should not put these rules under `*` selector, you might break other styling